src/main/kotlin/com/rooplor/classcraftbackend/services/AuthService.kt

```diff
@@ -45,6 +45,12 @@ class AuthService(
         return authentication?.name
     }
 
+    fun getAuthenticatedUserDetails(): User? {
+        val username = getAuthenticatedUser()
+        print(username)
```

Comment:
Can we remove this logging to avoid confuse log in future?

Reply:
Sorry I forgot to remove debug log.now removed already.